### ISSUE TITLE
nxdialog: autoreconf fails due to missing m4

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#792]
I see this on RHEL6:

```
cd nxdialog && autoreconf -vfsi && (./configure --prefix=/usr/local/nx/3.5.99 --libexecdir=/usr/local/nx/3.5.99/libexec --libdir=/usr/local/nx/3.5.99/lib64) && gmake
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
aclocal: couldn't open directory `m4': No such file or directory
autoreconf: aclocal failed with exit status: 1
```

`mkdir m4` solves the issue. We should include that directory or find a workaround.

**Comment by Ionic** [comment]
That should already be fixed through 3af59f80675c394ff2f24c1e2dccabcb5e383fac?

**Comment by uli42** [comment]
Right, I have totally missed that commit (forgot to pull before building...)
